torch/_inductor/ir.py

```diff
@@ -4594,6 +4594,32 @@ def __init__(  # type: ignore[no-untyped-def]
             allowed_prologue_inps if allowed_prologue_inps else OrderedSet()
         )
 
+        self.subgraph_inps: Optional[list[Optional[Union[IRNode, sympy.Expr]]]] = None
+        self.subgraph_outs: Optional[list[Optional[IRNode]]] = None
+
+    def get_free_symbol_uses(
```

Comment:
Maybe a doc block

Reply:
probably share the same [doc](https://github.com/pytorch/pytorch/blob/main/torch/_inductor/ir.py#L782-L804) as `get_free_symbol_uses` for other IRNode.